android/paper/src/main/java/com/facebook/react/viewmanagers/RNGestureHandlerButtonManagerDelegate.java

```diff
@@ -35,7 +35,7 @@ public void setProperty(T view, String propName, @Nullable Object value) {
         mViewManager.setEnabled(view, value == null ? true : (boolean) value);
         break;
       case "rippleColor":
-        mViewManager.setRippleColor(view, ColorPropConverter.getColor(value, view.getContext()));
```

Comment:
Removing `ColorPropConverter.getColor(value, view.getContext()` breaks `paper` builds when trying to set the color by it's name (`green` or `red`).
This only affects `paper`, because `fabric` has a different way of setting colors.